h5pt.go

```diff
@@ -20,10 +21,12 @@ import (
 // Table is an hdf5 packet-table.
 type Table struct {
 	Identifier
+	strings []*C.char
+	structs []unsafe.Pointer
 }
 
 func newPacketTable(id C.hid_t) *Table {
-	t := &Table{Identifier{id}}
+	t := &Table{Identifier{id}, make([]*C.char, 1), make([]unsafe.Pointer, 1)}
```

Comment:
I don't understand why we create the 2 slices with a length of 1.
it doesn't seem like we do `t.structs[0] = ... ` nor `t.strings[0] = ... `
it seems we always `append` to these slices.

maybe just leave them default-initialized?

Reply:
With the change you suggest below, this loses relevance :) But thanks for pointing out!